Modules/DevTools/TestFramework/TestRunner/src/ALTestTool.Page.al

```diff
@@ -303,12 +304,79 @@ page 130451 "AL Test Tool"
                     var
                         TestMethodLine: Record "Test Method Line";
                         TestSuiteMgt: Codeunit "Test Suite Mgt.";
+                        DeleteQuestion: Label 'Are you sure you want to Delete the Selected Lines?';
                     begin
+                        if GuiAllowed() then begin
+                            if not Confirm(DeleteQuestion, false) then
+                                exit;
+                        end;
+
                         CurrPage.SetSelectionFilter(TestMethodLine);
                         TestMethodLine.DeleteAll(true);
                         TestSuiteMgt.CalcTestResults(Rec, Success, Failure, Skipped, NotExecuted);
                     end;
                 }
+
+                action(SwitchRunCheckLines)
+                {
+                    ApplicationArea = All;
+                    Caption = '&Switch Run Check';
+                    Image = Change;
+                    Promoted = true;
+                    PromotedCategory = Process;
+                    PromotedIsBig = true;
+                    PromotedOnly = true;
+                    ToolTip = 'Switch Run Check on selected lines.';
+
+                    trigger OnAction()
+                    var
+                        TestMethodLine: Record "Test Method Line";
+                        SwitchQuestion: Label 'Are you sure you want to %1 the Selected Lines?';
```

Comment:
Placeholders for individual words are very hard to translate. The test runner isn't translated, so here it would be ok, but this is generally considered an anti-pattern.